src/sentry/static/sentry/app/views/organizationGroupDetails/groupTagValues.tsx

```diff
@@ -149,7 +149,7 @@ class GroupTagValues extends AsyncComponent<
             />
           </HeaderButtons>
         </Header>
-        <table className="table table-striped">
```

Comment:
This is the only place that uses this class so I’m moving those styles into this component